dev/io.openliberty.org.eclipse.microprofile/config.3.1.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.org.eclipse.microprofile.config.3.1; singleton:=true
+
+Import-Package: \
+  *
+
+Export-Package: \
+  org.eclipse.microprofile.config;version=3.0, \
+  org.eclipse.microprofile.config.inject;version=2.0, \
+  org.eclipse.microprofile.config.spi;version=3.0
```

Comment:
Will the 3.0 be changed to 3.1 at some point or will it stay at 3.0?

Reply:
Will stay as 3.0 ... almost all of the changes in MP Config 3.1 are in the TCK